README.md

```diff
@@ -1 +1 @@
-# fms-train
\ No newline at end of file
+# fms-pretrain
```

Comment:
we moved the repo name from fms-train to pretrain, should probably drop the 'fms-'

Reply:
Ok, will do